app/views/custom_fields/_index.html.erb

```diff
@@ -26,36 +26,100 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+          <th>
+            <div class="generic-table--sort-header-outer">
+              <div class="generic-table--sort-header">
+                <span>
+                  <%= CustomField.human_attribute_name(:name) %>
+                </span>
+              </div>
+            </div>
+          </th>
+          <th>
+            <div class="generic-table--sort-header-outer">
+              <div class="generic-table--sort-header">
+                <span>
+                  <%= CustomField.human_attribute_name(:field_format) %>
+                </span>
+              </div>
+            </div>
+          </th>
+          <th>
+            <div class="generic-table--sort-header-outer">
+              <div class="generic-table--sort-header">
+                <span>
+                  <%= CustomField.human_attribute_name(:is_required) %>
+                </span>
+              </div>
+            </div>
+          </th>
+          <% if tab[:name] == 'WorkPackageCustomField' %>
+            <th>
+              <div class="generic-table--sort-header-outer">
+                <div class="generic-table--sort-header">
+                  <span>
+                    <%= CustomField.human_attribute_name(:is_for_all) %>
+                  </span>
+                </div>
+              </div>
+            </th>
+            <th>
+              <div class="generic-table--sort-header-outer">
+                <div class="generic-table--sort-header">
+                  <span>
+                    <%=l(:label_used_by)%>
+                  </span>
+                </div>
+              </div>
+            </th>
+          <% end %>
+          <th>
+            <div class="generic-table--sort-header-outer">
+              <div class="generic-table--sort-header">
+                <span>
+                  <%=l(:button_sort)%>
+                </span>
+              </div>
+            </div>
+          </th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        <% (@custom_fields_by_type[tab[:name]] || []).sort.each do |custom_field| -%>
+          <tr>
+            <td><%= link_to h(custom_field.name), edit_custom_field_path(custom_field), lang: custom_field.name_locale %></td>
+            <td><%= Redmine::CustomFieldFormat.label_for(custom_field.field_format) %></td>
+            <td><%= checked_image custom_field.is_required? %></td>
+            <% if tab[:name] == 'WorkPackageCustomField' %>
+              <td><%= checked_image custom_field.is_for_all? %></td>
+              <td><%= l(:label_x_projects, :count => custom_field.projects.count) if custom_field.is_a? WorkPackageCustomField and !custom_field.is_for_all? %></td>
+            <% end %>
+            <td><%= reorder_links('custom_field', {:action => 'update', :id => custom_field}, method: :put) %></td>
+            <td class="buttons">
+              <%= delete_link custom_field_path(custom_field) %>
+            </td>
+          </tr>
+        <% end; reset_cycle %>
```

Comment:
the `reset_cycle` call is necessary only in combination with `cycle("odd", "even")` used for the colouring the table rows. As we don't do this any more, the call can be removed.